models/PostService.php

```diff
@@ -4,38 +4,45 @@
 
 namespace models;
 
+use app\Registry;
+
 class PostService
 {
     use TSingletone;
 
 
-    private $db;
+    private \PDO $db;
 
     protected function __construct()
     {
-        global $dbObject;
-        $this->db = $dbObject;
+
+        $this->db = Registry::get('dbObject');
     }
 
-    public function createPost(string $title, string $text, string $author): void
+    public function createPost(string $title, string $text, int $userId, int $categoryId): void
     {
-        $query = "INSERT INTO posts VALUES (NULL, ?,?,?,?)";
+        $query = "INSERT INTO posts VALUES (NULL, ?,?,?,?,?)";
         $statement = $this->db->prepare($query);
-        $statement->execute(array($title, $text, $author, time()));
+        $statement->execute(array($title, $text, $userId, $categoryId, time()));
     }
 
+
     public function getAllPosts(): array
     {
         $posts = array();

```

Comment:
```suggestion
        $posts = [];

```
сейчас принято писать коротко объявление массивов

Reply:
исправил, постарался найти везде где смог